.circleci/config.yml

```diff
@@ -74,6 +74,9 @@ run_linter: &run_linter
 test_spython: &test_spython
   name: Test Singularity Python (Singularity Version 2 and 3)
   command: |
+      cd ~/repo/spython/tests
```

Comment:
I'd go solely for python tests

Reply:
The client tests for basic command line usage are also needed.

Reply:
The better approach would be via python tests though. You can just import the module, set the argv and call main. Multiple testing approaches in 1 project are hard to handle and become harder over time

Reply:
We have tests in Python, I just wanted to mimic them on the command line. Look at it this way - if they get cumbersome we can always remove them. For the time being, they are very simple to check for return codes when interacting with the client, so I'd prefer to keep them until there is actual evidence of difficulty of having them.